### PR TITLE
Sanity check fitted for Qt 5

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyblocks/q/qt.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.run import run_cmd_qa
-
+from distutils.version import LooseVersion
```
Comment (boegel) [inline easybuild/easyblocks/q/qt.py]: this import should go at the top to keep it consistent with elsewhere:

```
import os
from distutils.version import LooseVersion
```

**Comment by boegel** [comment]
@hajgato: looks fine, but I guess this means only stuff that is well aware of Qt 5 can use Qt 5 as a dependency?

**Comment by hajgato** [comment]
@boegel: Yes. And some of them are requesting Qt 5 (Qt 4 does not work anymore), like `CGAL 4.7`

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Tested with both the Qt5 easyconfig in https://github.com/hpcugent/easybuild-easyconfigs/pull/2136 and an existing easyconfig for Qt 4.x, thanks @hajgato!

**Pull request merged**